config/locales/en.yml

```diff
@@ -510,6 +510,8 @@ en:
               invalid: Enter a valid contact telephone number
             candidate_instructions:
               blank: Enter some instructions for the candidate
+            experience_type:
+              blank: Select if it is an In school or Virtual experience
```

Comment:
I'm not sure about the capitalisation here. Could this be `Select an experience type`?

Reply:
That's based on the latest guidelines for radio error messages https://design-system.service.gov.uk/components/radios/#if-there-are-two-options-which-are-not-yes-and-no-

Reply:
I changed the error message to lowercase